cli_tools/common/utils/logging/service/loggables.go

```diff
@@ -0,0 +1,73 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package service
+
+import (
+	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
+	"strconv"
+	"strings"
+)
+
+// WorkflowToLoggable is a shim for a daisy workflow, exposing only those
+// fields that are pertinent to logging.
+func WorkflowToLoggable(wf *daisy.Workflow) Loggable {
+	return workflowLoggable{wf: wf}
+}
+
+type workflowLoggable struct {
+	wf *daisy.Workflow
+}
+
+func (w workflowLoggable) GetKeyValueAsString(key string) string {
```

Comment:
GetValue?

The rest is obvious from the func signature (although there would be a conflict with GetKeyValueAsInt64Slice). Also, `KeyValue` could be interpreted as a tuple, which this function doesn't return.

Reply:
sgtm